docker-compose.yml

```diff
@@ -18,6 +18,8 @@ services:
       GRYNOS_COURSES_UPDATE_INTERVAL: 3600000
       GRYNOS_COURSE_API_URL: https://colosseum.grynos.com/ilmoapix/v1/course/search?com=3,6&cgt=176
       POPULATE_SEED_DATA: 1
+      PAYMENT_RETURN_URL: https://vantaa-silverfox.herokuapp.com/api/payments/paymentReturn
```

Comment:
This Docker Compose configuration is used only to setup local environment and should not know anything about the Heroku deployments etc. In Heroku and production these things are set with variable variables and Docker Compose is not even used to run anything.